src/practitest_firecracker/core.clj

```diff
@@ -24,37 +24,45 @@
        (println (str ~module " elapsed time: " (/ (double (- (. System (nanoTime)) start#)) 1000000.0) " msecs")))
      ret#))
 
+(defn clean-tmp-folder [directory]
+  (doseq [path directory]
+    (delete-recursively! path)))
+
 (defn -main [& args]
   (let [{:keys [action options exit-message ok?]} (parse-args args)]
     (if exit-message
       (exit (if ok? 0 1) exit-message)
       (do
-        (remove-bom (file (first (:reports-path options))))
+        (doseq [report-path (:reports-path options)]
+          (remove-bom report-path))
         (let [client             (make-client (select-keys options [:email :api-token :api-uri :max-api-rate]))
-              reports            (parse-reports-dir (:reports-path options))
-              directory          (file (first (:reports-path options)))
+              directory          (map #(.getAbsolutePath (file "tmp" %)) (:reports-path options))
```

Comment:
Temp folder should be passed as a parameter. You might not have permissions to reports path and will not be able to create the "tmp" subfolder.